industry_benchmarks/input_structures/prepared_structures/merck/cmet/PREPARATION_DETAILS.md

```diff
@@ -0,0 +1,3 @@
+Protein structure from Schrodinger public benchmark set was reprepared with Protein preparation wizard with capped termini (replace hydrogens was switched off to fix protonation states and tautomers for histidines, optimize hbond assignments was turned off and all other settings kept at default).
```

Comment:
Are there any Maestro version numbers we can point to?

Reply:
Added the version number in the preparation details